src/client/routes/VaultDetail/index.tsx

```diff
@@ -124,14 +124,14 @@ export const VaultDetail = () => {
 
   return (
     <VaultDetailView>
-      <ViewHeader>
+      {/* <ViewHeader>
         <BackButton onClick={() => history.push(`/vaults`)}>{t('components.back-button.label')}</BackButton>
-      </ViewHeader>
+      </ViewHeader> */}
 
```

Comment:
Remove?

Reply:
Removed in 37090db0d7efc2706a2f90189ac28479f41cded6